config/personality/base.ini

```diff
@@ -12,3 +12,15 @@ apps_del = ${apps_add_defaults}
 [flatpak-remote-flathub]
 # Delete all default additions and explicitly build the list from scratch.
 apps_del = ${apps_add_defaults}
+apps_add =
+  org.learningequality.Kolibri
+
+[kolibri]
+install_channels =
+  # How to get started with Kolibri on Endless OS [23]
+  e8a879742b2249a0a4b890f9903916f7
+
+[kolibri-e8a879742b2249a0a4b890f9903916f7]
+include_node_ids =
+  # English [topic - 23]
+  3b909a18242c48208dbc49d06bc48162
```

Comment:
Perhaps this could live in `config/defaults.ini` too and then would be overridden for `es`, `ar` and `fr` with those versions' node IDs.

Reply:
Yeah, we could do that. That exact thing is commented out in config/defaults.ini right now, so we could uncomment that and delete this.

Reply:
Let's do that. If you are able to do the `config/personality/{es,ar,fr}.ini` overrides too that would be wonderful :)

Reply:
Okay, done :)